src/System.AppContext/src/ApiCompatBaseline.net46.txt

```diff
@@ -0,0 +1 @@
+MembersMustExist : Member 'System.AppContext.TargetFrameworkName.get()' does not exist in the implementation but it does exist in the contract.
```

Comment:
This was added in net462. It looks like we are getting the correct contract for a number of net46 compat runs based on this and other net46 baseline files. @venkat-raman251 can you please look to see if the correct older contract 4.0.0 in this case is getting passed for this build?